### DISCUSSION TITLE
Add a tooltip

### DISCUSSION THREAD

**Discussion by Assitan** [suren-atoyan/monaco-react#259] [General]
How can I implement this ?

https://microsoft.github.io/monaco-editor/playground.html#extending-language-services-hover-provider-example

Thanks

**Comment by suren-atoyan** [comment]
The equivalent of [this](https://microsoft.github.io/monaco-editor/playground.html#extending-language-services-hover-provider-example) with monaco-react would be [this one](https://codesandbox.io/s/hover-provider-7fuub?file=/src/App.js:3-129)

**Reply by Assitan** [reply]
Thank you
